voila/handler.py

```diff
@@ -187,6 +187,7 @@ def time_out():
             kernel_env[ENV_VARIABLE.VOILA_PREHEAT] = 'False'
             kernel_env[ENV_VARIABLE.VOILA_BASE_URL] = self.base_url
             kernel_env[ENV_VARIABLE.VOILA_SERVER_URL] = self.settings.get('server_url', '/')
+            kernel_env[ENV_VARIABLE.VOILA_APP_PORT] = str(port) if port else ''
```

Comment:
```suggestion
```

The value of `port` here is the port that `Voila` gets from the incoming request, not the port on which `Voila` is listening locally (for example in the case of `Voila` is served behind a proxy server). 
But we don't need to set the variable `ENV_VARIABLE.VOILA_APP_PORT` here since we are not in the preheated kernel mode.